src/acceptance/assets/app/go_app/Makefile

```diff
@@ -72,6 +72,10 @@ build: ./build/app ./build/manifest.yml
 	CGO_ENABLED='${CGO_ENABLED}' GOOS='linux' GOARCH='amd64' go build -o './build/app'
 	cp './app_manifest.yml' './build/manifest.yml'
 
+.PHONY: cf-build
+cf-build: ./build/app ./build/manifest.yml
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o './build/app'
+	cp './app_manifest.yml' './build/manifest.yml'
```

Comment:
What's this for? Isn't this the same as `build`?

Reply:
Removed in b62e9fbc9